frontend/src/components/NewShoot/NewShootInfrastructureDetails.vue

```diff
@@ -261,6 +244,43 @@ export default {
       'loadBalancerClassNamesByCloudProfileName',
       'floatingPoolNamesByCloudProfileNameAndRegion'
     ]),
+    validationErrors () {
+      const validationErrors = {
+        secret: {
+          required: 'Secret is required',
+          requiresCostObjectIfEnabled: () => {
+            const projectName = get(this.secret, 'metadata.projectName')
+            const project = projectFromProjectList()
+            const isSecretInProject = project.metadata.name === projectName
+
+            return isSecretInProject ? `${this.costObjectTitle} is required. Go to the ADMINISTRATION page to edit the project and set the ${this.costObjectTitle}.` : `${this.costObjectTitle} is required and has to be set on the Project ${toUpper(projectName)}`
```

Comment:
In case of own project, we could also make the `ADMINISTARTION page`a link to the project admin page and open the project settings dialog. But this is only optional.

Reply:
we can't have html in the error message, right? or do you mean we should display a link somewhere else?

Reply:
I don't know. maybe? Let's leave it as is for now

Reply:
👍